src/hb-unicode.h

```diff
@@ -240,16 +240,16 @@ typedef hb_script_t			(*hb_unicode_script_func_t)		(hb_unicode_funcs_t *ufuncs,
 										 hb_codepoint_t      unicode,
 										 void               *user_data);
 
-typedef hb_bool_t			(*hb_unicode_compose_func_t)		(hb_unicode_funcs_t *ufuncs,
-										 hb_codepoint_t      a,
-										 hb_codepoint_t      b,
-										 hb_codepoint_t     *ab,
-										 void               *user_data);
-typedef hb_bool_t			(*hb_unicode_decompose_func_t)		(hb_unicode_funcs_t *ufuncs,
-										 hb_codepoint_t      ab,
-										 hb_codepoint_t     *a,
-										 hb_codepoint_t     *b,
-										 void               *user_data);
+typedef bool			(*hb_unicode_compose_func_t)		(hb_unicode_funcs_t *ufuncs,
+									 hb_codepoint_t      a,
+									 hb_codepoint_t      b,
+									 hb_codepoint_t     *ab,
+									 void               *user_data);
+typedef bool			(*hb_unicode_decompose_func_t)		(hb_unicode_funcs_t *ufuncs,
+									 hb_codepoint_t      ab,
+									 hb_codepoint_t     *a,
+									 hb_codepoint_t     *b,
+									 void               *user_data);
```

Comment:
The second and third columns should stay aligned with the columns of the other typedefs.

Reply:
Done